spread/build/sbuild/task.yaml

```diff
@@ -38,6 +39,7 @@ execute: |
     MKSBUILD_OPTS=(
       mk-sbuild
       --distro ${DISTRO}
+      --debootstrap-include=ccache
```

Comment:
I *think* you could get away with passing `--ccache`, `--ccache-size`, and `--ccache-dir` arguments to `mk-sbuild`, **but** I don't think I've ever got that to work properly, and this does, so this comment is mostly for historical reasons in case this breaks in future and we want to try another approach.